### PR TITLE
Jk/bugfix/cumulus 3587 pull in express boom

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-3587](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3587)

Reviewers please note:   Units are (happily) functionally covered by the various failure/error output tests on the existing API.    

## Changes

* Brings in https://github.com/scottcorgan/express-boom with license so we can update sub-dependency/maintain without refactoring errors in API/etc wholesale

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: packages/api/lib/expressBoom.js
Comment (Jkovarik) [inline packages/api/lib/expressBoom.js:24]: Note: Given the nature of this fix, I avoided types/docstrings on the entire module, if we feel *strongly* we need to delve into express middleware typing as a meaningful exercise in this ticket that's worth a conversation.

File: bamboo/bootstrap-unit-tests.sh
Comment (Jkovarik) [inline bamboo/bootstrap-unit-tests.sh:17]: This is an unrelated cleanup that is now a separate PR https://github.com/nasa/cumulus/pull/3586 and should probably be removed in favor of #3586

File: bamboo/bootstrap-unit-tests.sh
Comment (paulpilone) [inline bamboo/bootstrap-unit-tests.sh:19]: This is a pretty broad `rm` command, no?

Comment (npauzenga) [reply bamboo/bootstrap-unit-tests.sh:19]: It looks like this is from a separate PR. This should either be removed from this one or https://github.com/nasa/cumulus/pull/3586 should be merged first.

In any case, I _think_ this is appropriate as we're saying effectively 'there should be no running docker containers'. We're clearing the table and then starting all of the appropriate containers (e.g. buid env, ftp server, ElasticSearch, etc.).

Comment (npauzenga) [reply bamboo/bootstrap-unit-tests.sh:19]: Ah I didn't see the comment above 😆 https://github.com/nasa/cumulus/pull/3583/files#r1488189445

File: packages/api/lib/expressBoom.js
Comment (npauzenga) [inline packages/api/lib/expressBoom.js:19]: Did we conclude that this was appropriate and doesn't require any kind of approval? It seems like yes, especially given the low complexity.

File: packages/api/lib/expressBoom.js
Comment (npauzenga) [inline packages/api/lib/expressBoom.js:30]: Did this come from us or is this eslint exception ported from the original?

**Review by npauzenga — APPROVED** [review]
This looks good to me. One comment just to clarify the licensing but otherwise 👍 

I'll let @paulpilone also approve 💪

**Pull request merged**
